CHANGELOG.md

```diff
@@ -123,6 +123,7 @@
 - `mezemoexporter`: Add user agent string to outgoing HTTP requests (#10470)
 - `prometheusreceiver`: Improve performance of metrics builder (#10546)
 - `transformprocessor`: Add functions for conversion of scalar metric types (`gauge_to_sum` and `sum_to_gauge`) (#10255)
+- `podmanreceiver`: Fetch containers stats one by one and add container image as metric attribute (#10421)
```

Comment:
Please move this to the "Unreleased" section

Reply:
Good catch! Fixed, thanks.